__mocks__/addAndRemove.js

```diff
@@ -86,11 +69,9 @@ class UserTask {
     if (TasksR[id].completed === true) {
       TasksR[id].completed = false;
       localStorage.setItem('TaskToday', JSON.stringify(TasksR));
-      console.log(TasksR);
     } else {
       TasksR[id].completed = true;
       localStorage.setItem('TaskToday', JSON.stringify(TasksR));
-      console.log(TasksR);
```

Comment:
please remove console.log() from your code following the best practices in javascript